pandas/tests/indexes/period/test_tools.py

```diff
@@ -101,6 +101,12 @@ def _get_with_delta(delta, freq='A-DEC'):
         tm.assert_index_equal(result.index, exp_index)
         assert result.name == 'foo'
 
+    def test_to_timestamp_freq(self):
+        idx = pd.period_range('2017', periods=12, freq="A-DEC")
+        result = idx.to_timestamp()
+        expected = pd.date_range("2017", periods=12, freq="AS-JAN")
+        tm.assert_index_equal(result, expected)
+
```

Comment:
Is this related to everything else?

Reply:
Had a bug in an earlier version. Wanted to make sure I didn't regress.